lib/phoenix/live_dashboard/live/page_live.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Phoenix.LiveDashboard.PageNotFound do
+  defexception [:message, plug_status: 404]
+end
+
+defmodule Phoenix.LiveDashboard.PageLive do
+  use Phoenix.LiveDashboard.Web, :live_view
+  import Phoenix.LiveDashboard.LiveHelpers
+
+  @impl true
+  def mount(%{"node" => _, "page" => page} = params, session, socket) do
+    if module = session[page] do
+      socket
+      |> assign_mount(String.to_existing_atom(page), params, session, true)
+      |> assign(:module, module)
+      |> maybe_apply_module(:mount, [params, session, :__socket__], &{:ok, &1})
```

Comment:
The socket is always the last parameter. So you can remove `:__socket__` here and instead always do `args ++ [socket]`.

Reply:
This is nice, I didn't notice, I'll update the code